docs/shocks.rst

```diff
@@ -1,140 +1,134 @@
+++++++++++++++++++++++++
 
-   - `stride`: Output image reduce factor
-   - `full_interval`: No. of steps between full restart snapshot written to file (-1 = disabled)
 
-- [simulation]
-   - `cfl`: The simulation time step in units of grid space
-   - `Nt`: The total number of laps in the simulation
-   
-- [problem]
-   - `Nspecies`: Number of species
-   - `delgam`: Temperature of plasma in units of mc^2 in bulk flow frame
-   - `gamma`: Lorentz factor of bulk flow
-   - `me`: electron mass-to-charge
-   - `mi`: positron/ion mass-to-charge
-   - `sigma`: Plasma magnetization parameter
-   - `npasses`: Number of current filter passes
-   
-Additionally, more advanced options include
+Grid and Dimensions
+------------------------
 
-   - `temp_ratio`: Ratio of ion temperature to electron temperature
-   - `bphi`: external B-field z angle
-   - `btheta`: external B-field x-y angle
-   - `wallgamma`: X velocity of the left piston wall (this makes the left wall ram into the plasma)
+Simulations are available in all three dimensions and you can set the size of the grid based on your needs. The following configuration parameters are found and can be edited in the ``.ini`` files.
 
-- [grid]
-   - `Nx`, `Ny`, `Nz`: No. of mesh tiles in x,y,z direction
-   - `NxMesh`/`NyMesh`/`NzMesh`: Size of tiles in x,y,z direction
-   
-Complete grid size is then `Ni*NiMesh`.
+Simulation framework:
 
-- [particles]
-   - `ppc`: Particles per cell per species
-   - `c_omp`: Skindepth resolution
+- ``[grid]``
+   - ``Nx``, ``Ny``, ``Nx`` : Number of tiles in corresponding directions
+   - ``NxMesh``, ``NyMesh``, ``NzMesh`` : Number of partitions each tile is divided into in each corresponding direction; internal grid.
+   - ``oneD``, ``twoD``, ``threeD`` : Forced dimensionality - `True` forces the dimension when other options are set to `False`.
+   - ``c_omp`` : Simulation skin depth resolution
 
-Additionally, more advanced options include
+Complete grid size in each dimension is determined as ``Ni*NiMesh``.
 
-   - `n_test_prtcls`: Number of test particles tracked in simulation
-   
 
+Relevant parameters to get the simulation running:
+--------------------------------------------------
 
-Running the simulation
-======================
-To run a shock simulation on Runko, use the following command:
+- ``[io]``
+   - ``outdir``, ``prefix``, ``postfix`` : Output directory name
+   - ``interval`` : Simulation output frequency for analysis in units of laps
 
-.. code-block:: bash
+- ``[simulation]``
+   - ``Nt`` : Maximum simulation time in units of laps
+   - ``npasses`` : Number of current filters
+   - ``mpi_track`` : Caterpillar cycle length
 
-   mpirun [-n no_of_cores] python3 pic.py --conf shock_mini.ini
+- ``[problem]``
+   - ``delgam`` : Upstream plasma temperature
+   - ``bpar``, ``bplan``, ``bperp`` : Magnetic fields in `x`, `y` and `z` directions accordingly. 
+   - ``sigma`` : Plasma magnetization parameter
+   - ``gamma`` : Upstream bulk flow speed
```

Comment:
Upstream bulk flow speed in units of Lorentz factor ``\Gamma = (1 - v^2/c^2)^{-1/2}`` (for ``gamma`` > 1) and 3-velocity ``\beta = v/c`` (for `gamma` < 1).